src/odemis/model/_metadata.py

```diff
@@ -122,12 +122,18 @@
 MD_POL_LHC = "lhc"  # (str) polarization analyzer position
 MD_POL_S0 = "S0"  # (str) Stokes parameter sample plane S0
 MD_POL_S1 = "S1"  # (str) Stokes parameter sample plane S1
-MD_POL_S2 = "S2"  # (str) Stokes parameter sample plane S1
-MD_POL_S3 = "S3"  # (st) Stokes parameter sample plane S1
+MD_POL_S2 = "S2"  # (str) Stokes parameter sample plane Ss
+MD_POL_S3 = "S3"  # (st) Stokes parameter sample plane S3
+MD_POL_S1N = "S1N"  # (str) Stokes parameter sample plane S1 normalized by S0
```

Comment:
Please fix the comments so that the Sx has the right number in S.
Also "st" -> "str"